.github/workflows/presubmit.yml

```diff
@@ -127,15 +140,21 @@ jobs:
         env:
           CI_PLATFORM: ${{ matrix.os }}
 
-      - name: Build
+      - name: Build & Install
         run: |
           TAICHI_REPO_DIR=`pwd`
           export PATH=$TAICHI_REPO_DIR/taichi-llvm/bin/:$PATH
           export CXX=clang++
-          python misc/ci_setup.py ci
+          python -m pip install -r requirements_dev.txt
+          cd python
+          git fetch origin master
+          TAICHI_CMAKE_ARGS=$CI_SETUP_CMAKE_ARGS python build.py build
+          cd ..
+          export NUM_WHL=`ls dist/*.whl | wc -l`
```

Comment:
Nit: you don't need to export NUM_WHL, simply setting it is enough in this case ;)